e2e/cypress-cucumber-preprocessor.config.js

```diff
@@ -3,5 +3,5 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 module.exports = {
-  stepDefinitions: 'e2e/features/step_definitions',
```

Comment:
I wasn't expecting to see the cucumber config change in a PR about i8ln - has something else crept in by mistake?

Reply:
Cypress tests were broken by the change to tsconfig to support react/jest. As part of getting the config working, I had changed to use cypress project to make the config cleaner (not needing to reference `e2e` in every path.

Reply:
I agree - I may have taken boy scout principle (leave it cleaner than you started) slightly too far with this